algorithm/matrix/matrix.go

```diff
@@ -0,0 +1,89 @@
+package matrix
+
+// Matrix is a one-dimensional matrix.
+type Matrix []float64
+
+// LineMatrix is a two-dimensional matrix.
+type LineMatrix [][]float64
+
+// PolygonMatrix is a there-dimensional matrix.
```

Comment:
A typo, "there-dimensional" should be "three-dimensional"

Reply:
take and done